src/pages/AuthPage/AuthPage.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { useStore, useDispatch } from "react-redux";
+import axios from 'axios';
+
+import { userSignUp, userLogin } from "../../ducks/authIntoSystem";
+
+import AuthForm from "../../components/AuthForm";
+
+import Img1 from '../../assets/auth-three-people-image.svg';
+import Img2 from '../../assets/auth-games-image.svg';
+import Img3 from '../../assets/auth-robot-image.svg';
+import Img4 from '../../assets/auth-mother-daughter-image.svg';
+import LogoSrc from '../../assets/logo.svg';
+
+import {
+    Auth, AuthBody, Collage, CollageImageWrapper, CollageImage,
+    AuthContent, AuthTitle, AuthFooter, AuthFooterItem, Logo
+} from "./AuthPage.style";
+
+const AuthPage = () => {
+    const store = useStore();
+    const dispatch = useDispatch();
+
+    const fetchData = (formData, authType) => {
+        switch (authType) {
+            case "login":
```

Comment:
instead of `switch ` operator, login and register button can call different async actions that will do requests.